include/libsemigroups/present.tpp

```diff
@@ -20,6 +20,76 @@
 // monoid presentations.
 
 namespace libsemigroups {
+  namespace presentation {
+
+    template <typename T>
+    T operator<<(T const& u, T const& w) {
+      T result(u);
+      result.insert(result.end(), w.cbegin(), w.cend());
+      return result;
+    }
+
+    template <typename T, typename = std::enable_if_t<IsWord<T>::value>>
+    void operator<<=(T& u, T const& v) {
+      u.insert(u.end(), v.cbegin(), v.cend());
+    }
+
+    template <typename T>
```

Comment:
Be good to have a version of this that changed its first argument in-place (i.e. so that the signature is `T&`)